spec/mailers/notifier_spec.rb

```diff
@@ -212,15 +212,8 @@
       expect(@mail["From"].to_s).to eq("\"#{@cnv.author.name} (diaspora*)\" <#{AppConfig.mail.sender_address}>")
     end
 
-    it "SUBJECT: has a snippet of the post contents" do
-      expect(@mail.subject).to eq(@cnv.subject)
-    end
-
-    it "SUBJECT: has 'Re:' if not the first message in a conversation" do
-      @cnv.messages << Message.new(text: "yo", author: eve.person)
-      @mail = Notifier.private_message(bob.id, @cnv.author.id, @cnv.messages.last.id)
-
-      expect(@mail.subject).to eq("Re: #{@cnv.subject}")
+    it "should include notifier translation data" do
```

Comment:
`it "should use a generic subject"` perhaps? Also you should be fine using the `I18n.t` alias below. And I would still love a spec actually asserting that `@cnv.subject` is not in `@mail.subject` :)